src/openms/include/OpenMS/KERNEL/MRMTransitionGroup.h

```diff
@@ -335,6 +335,12 @@ namespace OpenMS
         }
       }
 
+      for (typename std::vector<SpectrumType>::iterator pr_it = precursor_chromatograms_.begin(); pr_it != precursor_chromatograms_.end(); ++pr_it)
```

Comment:
This would definitely be very useful and I guess that extension of the current transition list data structures should be feasible.